katas/es6/language/generator/yield.js

```diff
@@ -0,0 +1,43 @@
+// Generators - Yield Expressions
+// To do: make all tests pass, leave the assert lines unchanged!
+
+describe('generator instance', () => {
```

Comment:
what do you think about 
`describe('generator - yield keyword is used to pause and resume a generator function', ...`
I get lots of inspiration from MDN https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/yield like here :)